cache/local_session.cpp

```diff
@@ -332,13 +334,27 @@ int local_session::remove_new(const struct dnet_id &id,
 	struct dnet_cmd_stats cmd_stats;
 	memset(&cmd_stats, 0, sizeof(struct dnet_cmd_stats));
 
+	n2_repliers repliers{
+		[](std::unique_ptr<n2_message> msg) {
+			return 0;
+		}, // on_reply
+		[](int err) {
+			return err;
```

Comment:
Is it guaranteed that the value returned from `on_reply_error` will be returned by `callbacks.n2_command_handler()`?

Reply:
Yes